src/ralph/admin/filters.py

```diff
@@ -3,13 +3,29 @@
 from datetime import datetime
 from functools import lru_cache
 
-from django.contrib.admin import SimpleListFilter
+from django.contrib.admin.filters import FieldListFilter
+from django.db import models
 from django.db.models import Q
 from django.utils.encoding import smart_text
 from django.utils.formats import get_format
 from django.utils.translation import ugettext_lazy as _
 
 
+class RalphFieldListFilter(FieldListFilter):
```

Comment:
Please think about changing register priorities to simplify our prefixed classes maybe?